core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3806,6 +3821,8 @@ private static String javadocLink(String fullyQualifiedClassname) {
     public static final String UNDERFS_VERSION = "alluxio.underfs.version";
     public static final String UNDERFS_OBJECT_STORE_BREADCRUMBS_ENABLED =
         "alluxio.underfs.object.store.breadcrumbs.enabled";
+    public static final String UNDERFS_OBJECT_STORE_DIRECT_PERSIST_ENABLED =
+        "alluxio.underfs.object.store.direct.persist.enabled";
```

Comment:
This makes is sound like this is a performance optimization, but in reality, it is an unsafe operation. Maybe we should use the word "unsafe" in this parameter, and make the lack of safety more clear in the description.

Reply:
I already put "unsafe" in the property description. do you feel strong about this @gpang ?

Reply:
The description does not make it explicit that it is an unsafe setting. I don't think `risk of multiple clients racing` makes it obvious that this is unsafe and can cause inconsistencies.

Reply:
renamed to `alluxio.master.unsafe.direct.persist.object.enabled`